build/build_apple_llm_demo.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+set -euo pipefail
+
+TOKENIZER="${1:-bpe}"
+ARTIFACTS_DIR_NAME="$2"
+
+APP_PATH="examples/demo-apps/apple_ios/LLaMA/LLaMA"
+
+if [[ "${TOKENIZER}" = "bpe" ]]; then
+  xcodebuild build-for-testing \
+    -project "${APP_PATH}.xcodeproj" \
+    -scheme LLaMAPerfBenchmark \
+    -destination platform="iOS" \
+    -allowProvisioningUpdates \
+    DEVELOPMENT_TEAM=78E7V7QP35 \
+    CODE_SIGN_STYLE=Manual \
+    PROVISIONING_PROFILE_SPECIFIER=iLLaMA \
+    CODE_SIGN_IDENTITY="iPhone Distribution" \
+    CODE_SIGNING_REQUIRED=No \
+    CODE_SIGNING_ALLOWED=No \
+    GCC_PREPROCESSOR_DEFINITIONS="DEBUG=1 ET_USE_TIKTOKEN=0"
+else
+  xcodebuild build-for-testing \
+    -project "${APP_PATH}.xcodeproj" \
+    -scheme LLaMAPerfBenchmark \
+    -destination platform="iOS" \
+    -allowProvisioningUpdates \
+    DEVELOPMENT_TEAM=78E7V7QP35 \
+    CODE_SIGN_STYLE=Manual \
+    PROVISIONING_PROFILE_SPECIFIER=iLLaMA \
+    CODE_SIGN_IDENTITY="iPhone Distribution" \
+    CODE_SIGNING_REQUIRED=No \
+    CODE_SIGNING_ALLOWED=No
+fi
+
+# The hack to figure out where the xctest package locates
+BUILD_DIR=$(xcodebuild -showBuildSettings -project "$APP_PATH.xcodeproj" -json | jq -r ".[0].buildSettings.BUILD_DIR")
+
+# Prepare the demo app
+MODE="Debug"
```

Comment:
Make it Release by default?

Reply:
I try it out and think let's just stick with debug mode for now as it's the default one that xcodebuild uses, and it's also referred to in the test spec.  Maybe there is some benefit when it comes to benchmark in release build, but it's hard to say now